tasks/main.yml

```diff
@@ -60,6 +33,33 @@
       {{ efi_path | ternary(__bootloader_uefi_conf_dir ~ 'user.cfg',
       '/boot/grub2/user.cfg') }}
 
+- name: Get stat of {{ __bootloader_default_grub }}
+  stat:
+    path: "{{ __bootloader_default_grub }}"
+  register: __bootloader_grub_stat
+
+- name: >-
+    Update boot loader timeout configuration in {{ __bootloader_default_grub }}
+  replace:
+    path: "{{ __bootloader_default_grub }}"
+    regexp: '^GRUB_TIMEOUT=.*'
+    replace: 'GRUB_TIMEOUT={{ bootloader_timeout }}'
+    mode: "{{ __bootloader_grub_stat.stat.exists |
+      ternary(__bootloader_grub_stat.stat.mode, '0644') }}"
```

Comment:
Does one apply to the current kernel, and the other applies to new kernels?

Reply:
This is general, not applied to any kernel but to the system generally. This timeout happens before you use any kernel, actually.